.gitignore

```diff
@@ -7,6 +7,4 @@ obj/
 !./.vscode/settings.json
 !./.vscode/tasks.json
 !./.vscode/launch.json
-!./.vscode/extensions.json
-
-appsettings.json
\ No newline at end of file
+!./.vscode/extensions.json
```

Comment:
appsettings.json should not be commited (has app ID and secret)

Reply:
You'll want to purge the appsettings.json files from the repo since they've been commited.

Reply:
Removed **appsettings.json** from all demos & added entry to **.gitignore** to ignore in the future. We left placeholders in there to address this, but removed regardless.